types/bech32/index.d.ts

```diff
@@ -0,0 +1,14 @@
+// Type definitions for bech32 1.1
+// Project: https://github.com/bitcoinjs/bech32
+// Definitions by: Satana Charuwichitratana <https://github.com/micksatana>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+/// <reference types="node" />
+
+export function decode(str: string, LIMIT?: number): { prefix: string, words: number[] };
+
+export function encode(prefix: string, words: number[], LIMIT?: number): string;
+
+export function fromWords(words: number[]): number[];
+
+export function toWords(bytes: Buffer): number[];
```

Comment:
`s/Buffer/number[]` IMHO,  nothing specifies it needs to be a `Buffer` in the actual code

Reply:
Hi @dcousens,

Thank you very much for the review. This PR already merged so I open a new one at https://github.com/DefinitelyTyped/DefinitelyTyped/pull/22690